website/api/controllers/unsubscribe-from-newsletter.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+
+
+  friendlyName: 'Unsubscribe from newsletter',
```

Comment:
I think renaming this would be smart: 

> Unsubscribe from all newsletters

OR merge it with your upsert action, and use a separate request param `unsubscribeFromAll: true`

If set, I think it should clear all of the boolean flags too, since that way most of the time in your code you only have to check the boolean flags.

Reply:
I renamed this action `unsubcribe-from-all-newsletters.js`